src/main/java/uk/ac/ebi/tsc/portal/config/ECPApplicationProperties.java

```diff
@@ -0,0 +1,19 @@
+package uk.ac.ebi.tsc.portal.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "ecp.roles.deployments")
+public class ECPApplicationProperties {
+
+    private String view;
+
+    public String getView() {
+        return view;
+    }
+
+    public void setView(String view) {
+        this.view = view;
+    }
+}
```

Comment:
Some inconsistencies here in naming/scoping. You call a class `ECPApplicationProperties` and it only captures "ecp.roles.deployments", so the class name should reflect that ("DeploymentRolesConfiguration" would make bigger sense). Also consider, if you want that granularity of a config. Maybe it is just OK to have: "ecp.roles" instead capturing all roles in the entire application. Then the class should be sth like `RolesConfiguration`.

Also instead of roles we might use AAP language, where the role is called a Domain.

I would probably go for a class: AAPDomainsConfiguration; property `ecp.aap.domains.deployments.view` and a capture `ecp.aap.domains` capture for the configuration

Reply:
Fixed